docs/pages/reference/networking.mdx

```diff
@@ -206,3 +215,33 @@ Read more in our [TLS Routing](../architecture/tls-routing.mdx) guide.
 
 </TabItem>
 </Tabs>
+
+### Agent ports
+
+For Teleport processes running agents, e.g., instances of the SSH Service,
+Kubernetes Service, and other services that protect resources in your
+infrastructure, there is no need to open ports on the instance to the public
+internet. 
+
+Teleport agents dial the Teleport Proxy Service to establish a reverse tunnel.
+Client traffic flows via the Proxy Service to the agent, and the agent forwards
+traffic to resources in your infrastructure.
+
+Some Teleport services listen for traffic to one of their proxied resources,
+meaning that you can expose ports on that service's host directly to clients.
```

Comment:
Perhaps add some explanation that one reason you may want to know/expose these ports is so that you can connect to these instances directly in the event of a proxy service outage. It's certainly not something I'd expect us to encourage most customers to do - but I think it provides some explanation of in what situations you might want to do this.